edge-hub/src/Microsoft.Azure.Devices.Routing.Core/endpoints/StoringAsyncEndpointExecutor.cs

```diff
@@ -48,7 +48,7 @@ public class StoringAsyncEndpointExecutor : IEndpointExecutor
 
         public EndpointExecutorStatus Status => this.machine.Status;
 
-        public async Task Invoke(IMessage message)
+        public async Task Invoke(IMessage message, uint priority, uint timeToLiveSecs)
```

Comment:
To be implemented in next PR?

Reply:
yes, this PR just plumbs everything down to this layer.

---
In reply to: [376244577](https://github.com/Azure/iotedge/pull/2469#discussion_r376244577) [](ancestors = 376244577)